app/src/main/java/com/example/ithaca_transit_android_v2/states/SearchState.kt

```diff
@@ -19,11 +19,16 @@ class EmptyInitClickState : SearchState()
  */
 class InitClickState : SearchState()
 
-// Immediately following InitClickState or EmptyInitClickState when user begins typing.
+// Immediately following InitClickState or EmptyInitClickState when user begins typing
 class InitSearchState(
+    val searchText:String
+) : SearchState()
+
+// Immediately following InitSearchState after network requests are made
+class InitLocationsSearchState(
     val searchText: String,
     // Network call to all locations that match the search text
-    val searchedLocations: List<Location>
+    val searchedLocations: List<Location>?
```

Comment:
reason why this is nullable?

Reply:
Backend response gives us null. Like if you query "Carl B" response is null (I think)

Reply:
is this an intended behavior from backend?